data-prepper-plugins/opensearch/src/main/resources/index-template/logs-otel-v1-index-standard-template.json

```diff
@@ -0,0 +1,140 @@
+{
+  "version": 1,
+  "template": {
+    "mappings": {
+      "date_detection": false,
+      "_source": {
+        "enabled": true
+      },
+      "properties": {
+        "droppedEventsCount": {
+          "type": "integer"
+        },
+        "attributes": {
+          "type": "object"
+        },
+        "instrumentationScope": {
+          "properties": {
+            "name": {
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword",
+                  "ignore_above": 128
+                }
+              }
+            },
+            "version": {
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword",
+                  "ignore_above": 256
+                }
+              }
+            },
+             "attributes": {
+               "type": "object"
+             },
+             "droppedEventsCount": {
+               "type": "integer"
+             }
+          }
+        },
+        "resource": {
+          "properties": {
+             "attributes": {
+               "type": "object"
+             },
+             "droppedEventsCount": {
+               "type": "integer"
+             }
+          }
+        },
+        "severity": {
+          "properties": {
+            "number": {
+              "type": "long"
```

Comment:
This is an unsigned integer in the range 0-25. I think integer would be a sufficient field type.